### PR TITLE
adding favorite cupcake flavor topic

### PR DESCRIPTION
**Pull request by ThePotatoGuy**
Co-authored-by: @karrigantaylor

# TODO
- [x] sprite codes
- [x] categories

# Key Changes
* adds favorite cupcake flavor topic

### CODE REVIEW

**Review by jmwall24 — DISMISSED** [review]

**Review by multimokia — APPROVED** [review]

**Comment by multimokia** [comment]
@ThePotatoGuy awaiting rev of changes.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -17740,3 +17741,29 @@ label monika_hedonism:
...
+
+label monika_cupcake_favorite:
+    m 1rta "Hmm, I'm not sure I really have one..."
+    m 1hub "I like all sorts of different kinds, so it's hard to choose just one!"
+    m 3ekd "I think I've mentioned before how much I miss Natsuki's cupcakes..."
+    m 3eua "One time she made this really strange mint chocolate chip flavored cupcake...{w=0.3}it had mint flavored frosting with chocolate sprinkles and a chocolate cake base."
+    m 4rksdlb "It was one of the strangest things I've ever tasted ahaha!"
+    m 2eksdlb "It didn't really taste at all like how mint chocolate chip ice cream tastes, instead it sorta tasted like toothpaste!"
+    m 2ekp "It was kinda disappointing...{w=0.3}I was expecting it to be my favorite cupcake flavor."
+    m 7eka "Oh well, it was nice she tried to make me something unique I would like...{w=0.3}despite her tough exterior, she could be really sweet~"
```
Comment (lunulae) [inline Monika After Story/game/script-topics.rpy:17768]:
```suggestion
    m 7eka "Oh well, it was nice that she tried to make me something unique I would like...{w=0.3}despite her tough exterior, she could be really sweet~"
```

**Review by karrigantaylor — DISMISSED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -17809,3 +17810,29 @@ label monika_conventions:
...
+            action=EV_ACT_UNLOCK
+        )
+    )
+
+label monika_cupcake_favorite:
+    m 1rta "Hmm, I'm not sure I really have one..."
+    m 1hub "I like all sorts of different kinds, so it's hard to choose just one!"
+    m 3ekd "I think I've mentioned before how much I miss Natsuki's cupcakes..."
+    m 3eua "One time she made this really strange mint chocolate chip flavored cupcake...{w=0.3}it had mint flavored frosting with chocolate sprinkles and a chocolate cake base."
+    m 4rksdlb "It was one of the strangest things I've ever tasted ahaha!"
```
Comment (MrVavlo) [inline Monika After Story/game/script-topics.rpy:17834]: There's missed comma in this line before "ahaha".

Comment (ThePotatoGuy) [reply Monika After Story/game/script-topics.rpy:17834]: this is already merged so put this on the typos issue.

**Pull request merged**